### ISSUE TITLE
White outline around sprites in all examples.

### ISSUE THREAD

**Issue by nikitablack** [bevyengine/bevy#7268]
## Bevy version

Web examples

## What you did

Opened examples in Chrome (Version 108.0.5359.124 (Official Build) (64-bit)) and Firefox (108.0.2 (64-bit)). All scenes have problems with rendering apparently antialiased objects - both 2d and 3d:

![image](https://user-images.githubusercontent.com/6390092/213206092-3f0a73b0-adbe-49b4-bec6-20ec53572b8f.png)

![image](https://user-images.githubusercontent.com/6390092/213206896-72c57b94-b9cb-46ff-961d-0c7b0705fa2d.png)

![image](https://user-images.githubusercontent.com/6390092/213207268-210b6b36-405e-4a7c-b3e3-42d4eeabfc93.png)

I am running on Linux Ubuntu 18.04, Nvidia Quadro P1000, Driver Version: 525.60.13

**Comment by mockersf** [comment]
maybe related to #3593

**Comment by nikitablack** [comment]
Indeed. I tried to find a similar issue, but didn't see this one. In my case, I don't know the version and can't change samples count.

**Comment by hymm** [comment]
This may be fixed in https://github.com/bevyengine/bevy/pull/7356. @nikitablack are you able to test?

**Comment by nikitablack** [comment]
I tested it. It is the same. The interesting thing I noticed though is that _some_ examples look correct:

![image](https://user-images.githubusercontent.com/6390092/215710596-cff7600e-3ae9-4894-aaa6-b46c2e4eb6b5.png)

This is a screenshot from the UI/text example.
